wire-ios-notification-engine/Sources/NotificationSession.swift

```diff
@@ -387,7 +387,10 @@ extension NotificationSession: PushNotificationStrategyDelegate {
             events,
             publicKeys: try? earService.fetchPublicKeys()
         )
-        processDecodedEvents(decodedEvents)
+
+        await context.perform { [self] in
+            self.processDecodedEvents(decodedEvents)
+        }
```

Comment:
question: out of curiosity: is it necessary to retain self explicitly `[self]` or just something you prefer to do?

Reply:
I think it's a mistake from two possibilities:

```
   await context.perform { [self] in
            processDecodedEvents(decodedEvents)
   }
```

and

```
await context.perform { 
            self.processDecodedEvents(decodedEvents)
}
```